app/routes/lending/LendableObjectsList.tsx

```diff
@@ -45,6 +48,75 @@ const parseQuery = (search: string): Query => {
   };
 };
 
+enum status {
+  PENDING = "pending",
+  APPROVED = "approved",
+  DENIED = "denied",
+}
+
+const ApprovedFlag = () => {
+  return (
+    <div className={cx(styles.status, styles.approved)}>
+      <Icon name={'checkmark-circle-outline'} size={30} />
+      Godkjent!
+    </div>
+  )
+}
+
+const PendingFlag = () => {
+  return (
+    <div className={cx(styles.status, styles.pending)}>
+      <Icon name={'time-outline'} size={30} />
+      Venter på svar
+    </div>
+  )
+}
+
+const DeniedFlag = () => {
+  return (
+    <div className={cx(styles.status, styles.denied)}>
+      <Icon name={'close-circle-outline'} size={30} />
+      Avslått..
```

Comment:
Could these share the same styling as the pills on the event attendees admin page?

Reply:
I agree. Because they are so small on the attendees admin page I will slightly alter it to fit better in this setting